config/routes.rb

```diff
@@ -1,8 +1,9 @@
 Myflix::Application.routes.draw do
-  root to: "pages#front"
+  root to: 'pages#front'
 
-  get 'home', to: "videos#index"
+  get 'home', to: 'videos#index'
   get 'register', to: 'users#new'
+  get 'register/:token', to: 'users#new_with_invitation_token', as: 'register_with_token'
```

Comment:
Line is too long. [89/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.